schemas/event.yml

```diff
@@ -71,16 +71,18 @@
             This value is used to indicate that this event that a numeric measurement
             was taken at given point in time.
 
-            Examples include CPU utilization, memory usage, or a vulnerability scan result.
+            Examples include CPU utilization, memory usage, or device temperature.
 
             Metric events are often collected on a predictable frequency, such as once
             every few seconds, or once a minute.
         - name: state
           description: >
             This value is similar to metric, except that the entity being measured does not
-            provide a numeric metric value, but rather one of a fixed set of conditions or states.
+            provide a numeric metric value, but rather one of a fixed set of categorical values
+            that represent conditions or states.
+
             For example a periodic event reporting a "fin_wait" state of a TCP connection
-            on a host might use `event.type:state`.
+            on a host, or the result of a vulnerability scan on a host, could use `event.type:state`.
```

Comment:
I don't think we should be using vulnerability scan as an example for `event.kind` state. I think the "state" aspect of a vulnerability scan isn't direct enough.

Clearer examples of state events could be the state of an Elasticsearch cluster (green/yellow/red). Or the state of a TCP socket in general. There's "fin_wait", but there's also all of the other possible states: open, closed, etc.

Reply:
maybe a couple of examples? e.g. hosts state of vulnerability to a given attack, hosts tcp socket state, file state open/locked? Cluster state seems like it would be a periodically measured thing (metric), but I could be basing that on an core understanding of "cluster state" thru the lens of  e.g. cisco ASA HA polling, or MS clustering

Reply:
@webmat I'd really like to keep the vulnerability example, as ECS is otherwise silent on how to categorize events that populate the `vulnerability.*` field set, and I've heard this question asked several times.  I think that `event.kind:state` is the correct kind categorization for a vulnerability check performed on a single host.  i.e., the "state" result will be "vulnerable" or "not vulnerable."

I've added a few more examples from yours and @dainperkins suggestions to help clarify.

> Although I'm not sure I like the approach in the point above. I think all state events should be in event.kind:state, and whether it's a transition or an informational update should be determined via another field.

I disagree. I'd be worried that this could cloud the definition of event.kind"event".   As proposed, we now have an easily summarized anecdote to distinguish the three. 

 _Regarding `event.kind:`, if something happened: use `event`; if something numerical is measured: use `metric`; if something categorical is measured: use `state`._

Accordingly, I've clarified that state changes should be categorized as `event.kind:"event"`

Reply:
Ok, I like the resulting description for "state", it makes a lot of sense.

I'm back with you and @andrewstucki, let's have state changes be "events" 👍